engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/OauthTransform.kt

```diff
@@ -0,0 +1,87 @@
+package graphql.nadel.enginekt.transform
+
+import graphql.GraphQLError
+import graphql.introspection.Introspection
+import graphql.nadel.Service
+import graphql.nadel.ServiceExecutionResult
+import graphql.nadel.enginekt.NadelExecutionContext
+import graphql.nadel.enginekt.blueprint.NadelOverallExecutionBlueprint
+import graphql.nadel.enginekt.transform.query.NadelQueryTransformer
+import graphql.nadel.enginekt.transform.query.QueryPath
+import graphql.nadel.enginekt.transform.result.NadelResultInstruction
+import graphql.nadel.enginekt.transform.result.json.JsonNodeExtractor
+import graphql.nadel.enginekt.util.queryPath
+import graphql.normalized.ExecutableNormalizedField
+import graphql.validation.ValidationError
+import graphql.validation.ValidationErrorType
+
+class OauthTransform : NadelTransform<GraphQLError> {
```

Comment:
this class is here only for test purposes, I don't want to keep it, just something to help verify that a custom transformation works